src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.main__container {
+  display: grid;
+  gap: 46px 48px;
+  padding: 50px 40px;
+  place-content: center;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  border: 1px solid $color-gray;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    width: 100%;
+    color: $text-main-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 0;
+    margin-bottom: 4px;
+    transition: color 300ms ease;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+  &__code {
+    color: $text-secondary-color;
```

Comment:
Ensure that the `$text-secondary-color` variable is defined in your SCSS files. If it's not defined, this could lead to a compilation error.